### PR TITLE
Fix docker build

### PR DESCRIPTION
**Pull request by yoooou**

### CODE REVIEW

**Comment by sindack** [comment]
retest this please

File: Dockerfile [hunk 1]
```diff
@@ -75,6 +75,10 @@ COPY --from=build /c3po/pcrf/tssf/bin /bin
 
 ## Stage hssdb
 FROM cassandra:2.1.20 as hssdb
```
Comment (krsna1729) [inline Dockerfile]: Seems like changing to 2.1 works without the workaround. This way we get critical fixes too?
```suggestion
FROM cassandra:2.1 as hssdb
```

File: Dockerfile [hunk 2]
```diff
@@ -75,6 +75,10 @@ COPY --from=build /c3po/pcrf/tssf/bin /bin
...
+# We need the following workaround to ignore http://www.apache.org/dist/cassandra/debian
+# listed in /etc/apt/sources.list.d/cassandra.list which redirects to https. Once
+# apt-transport-https is installed we can continue with apt-get update and install
+RUN apt-get update -o Dir::Etc::SourceParts= && apt-get -y install apt-transport-https
```
Comment (krsna1729) [inline Dockerfile]:
```suggestion
```

File: Dockerfile (hunk 1 above)
Comment (yoooou) [reply Dockerfile]: Sure. I'll close this PR and open another one

Comment (yoooou) [reply Dockerfile]: Actually I'll just commit your suggestions

**Review by sindack — APPROVED** [review]

**Comment by yoooou** [comment]
retest this please

**Pull request merged**